javascript/imageviewer.js

```diff
@@ -136,7 +136,14 @@ function setupImageForLightbox(e) {
     var event = isFirefox ? 'mousedown' : 'click';
 
     e.addEventListener(event, function(evt) {
-        if (!opts.js_modal_lightbox || evt.button != 0) return;
+        if (!opts.js_modal_lightbox || evt.button != 0) {
+
+            if (evt.button < 2) {
```

Comment:
Incidentally also covering `!opts.js_modal_lightbox`, or perhaps I should `evt.button == 1` explicitly?